backend/uclapi/common/decorators.py

```diff
@@ -305,7 +305,18 @@ def wrapped(request, *args, **kwargs):
                     raise UclApiIncorrectDecoratorUsageException
 
             # In any case, a token should be provided
-            token_code = get_var(request, 'token')
+            if 'HTTP_AUTHORIZATION' in request.META:
+                token_code = request.META['HTTP_AUTHORIZATION']
+
+                # To remove the starting BEARER or AUTHENTICATION if provided
+                if ' ' in token_code:
+                    token_code = token_code.split(' ')[1]
```

Comment:
```suggestion
                token_code = token_code.lstrip('Bearer: ')
```
Firstly, this will match anything before. Secondly, I thought only `Bearer: <token>` was valid, with a quick google I didn't see any mention of `Authentication: <token>`.

Reply:
Yes you are absolutely right, just read through the [RFC-6750](https://tools.ietf.org/html/rfc6750) and not sure where I got the idea it could be any string.

Reply:
I believe [RFC6750 section 2.1](https://datatracker.ietf.org/doc/html/rfc6750#section-2.1) specifies a single space character between `Bearer` and the actual token.